src/blenderbim/blenderbim/bim/__init__.py

```diff
@@ -71,6 +71,7 @@
     "covetool": None,
     "augin": None,
     "debug": None,
+    "ifcgit": None,
```

Comment:
The module load order determines the panel draw order so maybe put it above "debug" at least :)

Reply:
The IFC Git panel gets added as a sub-panel of IFC Project Info, so it appears a bit further up the Scene Properties anyway